src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/StackUtilities.cs

```diff
@@ -17,23 +17,22 @@ internal enum StackFormat
 
     internal static class StackUtilities
     {
-        public static Models.CallStack TranslateCallStackToModel(CallStack stack, NameCache cache, bool methodNameIncludesGenericParameters = true)
+        public static Models.CallStack TranslateCallStackToModel(CallStack stack, NameCache cache, bool parameterTypesSupported = true)
```

Comment:
Can we change the parameter to indicate what it's actually doing? Something like `ensureParameterTypeFieldsNotNull`. I don't know what effect `parameterTypesSupported` is supposed to have without reading the method body.